src/treemodel/delta/transform.js

```diff
@@ -0,0 +1,170 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+import OT from '../operation/transform.js';
+
+import AttributeDelta from './attributedelta.js';
+import InsertDelta from './insertdelta.js';
+import MergeDelta from './mergedelta.js';
+import MoveDelta from './movedelta.js';
+import RemoveDelta from './removedelta.js';
+import SplitDelta from './splitdelta.js';
+import Delta from './delta.js';
+
+import arrayUtils from '../../lib/lodash/array.js';
+import utils from '../../utils.js';
+
+const baseDeltas = [ AttributeDelta, InsertDelta, MoveDelta, RemoveDelta ];
```

Comment:
If it only used to get priority should not it be called `priorities`? Or would not it be better if deltas have priorities?